src/zenml/cli/integration.py

```diff
@@ -175,10 +175,18 @@ def export_requirements(
     help="Force the installation of the required packages. This will skip the "
     "confirmation step and reinstall existing packages as well",
 )
+@click.option(
+    "--upgrade",
+    "-u",
+    "upgrade",
+    is_flag=True,
+    help="Upgrade the packages if they are already installed.",
+)
```

Comment:
I think it'd be nice for the user to get upgrades for only the integrations that they install. (i.e instead of having to specify each one individually). Not sure what the interface / CLI command would look like for that, though... Maybe there should be another flag to enable that?